### DISCUSSION TITLE
General Discussion

### DISCUSSION THREAD

**Discussion by pravee42** [pravee42/Hacktoberfest2022-python#82] [General]
Dear developers, this is a general discussion here all contributors of this repo can chat with each other

if are new to opensource and need help to contribute opensource 
join this discord server and can ask any doubt there https://discord.gg/pSup8ZZV

**Comment by pravee42** [comment]
hi @Llambi we appreciate your work in algorithms 
#66 #69 #68 #67

**Reply by Llambi** [reply]
It's been a pleasure, you're welcome 😊

**Comment by pravee42** [comment]
hi @MarkL50,
we appreciate you work on Python GUI to make an egg catching game

**Comment by pravee42** [comment]
HI @Indrahas,
your work on PR #118 was nice idea we appreciate it

**Comment by pravee42** [comment]
hi @anantpatel31,
Your work on open-cv was amazing and it will help a lot of users who currently learning open-cv and AI.
as a student of AI/ML, I will recommend your project to my friends who will learn to do projects

**Reply by anantpatel31** [reply]
Thank You @pravee42

**Comment by pravee42** [comment]
Hi @shyanukant,
we appreciate your work on automatic email sender using SMTP python for hacktoberfest 2022

**Reply by shyanukant** [reply]
> Hi @shyanukant, we appreciate your work on automatic email sender using SMTP python for hacktoberfest 2022

Thank you @pravee42
